content/en/apps/guides/data/users-bulk-load.md

```diff
@@ -126,3 +126,72 @@ You will need to manually add these new places to the spreadsheet so that you ca
 In this case "Site Dieco" is a CHW place, so we'll go to the "contact.c62_chw_VLOOKUP" worksheet.
 Add some new rows at the bottom (item 1), enter the new place name in column A (item 2) and paste the UUID in column B (item 3)
 ![open your existing spreadsheet](adding-places-existing-spreadsheet.png)
+
+## Trouble shooting
+
+### "Wrong type, this is not a person."
+
+If you have miss-matched contact types, you will get an error upon import:
+
+> **Wrong type, this is not a person.**
+
+
+As of CHT 3.7.0, you're [allowed to declare different contact types]({{< relref "apps/reference/app-settings/hierarchy#app_settingsjson-contact_types" >}}) in your `app_settings.json`. If you have added a `.contact_types[]` stanza to your JSON, you will need to update the automatic value of the `contact.contact_type` column.  The default value is:
```

Comment:
```suggestion
As of CHT 3.7.0, you're [allowed to declare different contact types]({{< relref "apps/reference/app-settings/hierarchy#app_settingsjson-contact_types" >}}) in your `app_settings.json`. If you have populated the `.contact_types[]` property in your JSON, you will need to update the automatic value of the `contact.contact_type` column.  The default value is:
```

I had never seen the word `stanza` before but it's pretty satisfying to say out loud :joy:  
Since the audience of these docs are usually not native English speakers, I think it would be safer to fall back to more common words

Reply:
Ah - it is a great word!  I'll go with the more common suggestion - thanks!